components/brave_ads/core/internal/segments/segment_util.cc

```diff
@@ -122,4 +124,9 @@ bool ShouldFilterSegment(const std::string& segment) {
   return iter != filtered_segments.cend();
 }
 
+void NormalizeSegment(std::string& segment) {
+  base::ReplaceChars(StripNonAlphaNumericCharacters(segment), " ", "",
+                     &segment);
+}
```

Comment:
We talked about that one — just making sure we don't forget to revert this (as the input and output here are not necessarily of the same size).

Reply:
Reverted as discussed